plugin/install.go

```diff
@@ -105,12 +110,44 @@ func doPluginInstall(c *cli.Context) error {
 	if err != nil {
 		return errors.Wrap(err, "Failed to install plugin while making a download URL")
 	}
+
+	canUpgrade := true
+	meta, err := newMetaDataStore(pluginDir, it)
+	if err != nil {
+		if err == errDisaleMetaDataStore {
+			canUpgrade = false
+		} else {
+			return errors.Wrap(err, "Failed to prepare meta data store")
+		}
+	}
+
+	overwrite := c.Bool("overwrite")
+	if canUpgrade && c.Bool("upgrade") {
+		releaseTag, err := meta.load("release_tag")
+		if err != nil {
+			return errors.Wrap(err, "Failed to load release_tag")
+		}
+		if releaseTag == it.releaseTag {
+			logger.Log("", fmt.Sprintf("release_tag %s already exists. Skip installing for now", it.releaseTag))
```

Comment:
`releaseTag == it.releaseTag` means same version is **installed**.  So `release_tag %s is already installed.` is better.

Reply:
or `release_tag %s of specified plugin is already installed.` to make it easy to understand for users.

Reply:
fixed in fc3b81d3190e60262635d9a787670c6e0eb8a84b